dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -250,6 +276,35 @@ def _run_tests_in_pool(
     skip_docker_compose_down: bool,
 ):
     escaped_tests = [test.replace("[", "\\[") for test in tests_to_run]
+    isolated_test_types = []
+    for isolated_test_type in ISOLATED_TEST_CASES:
+        if isolated_test_type in tests_to_run:
+            isolated_test_types.append(isolated_test_type)
+            tests_to_run.remove(isolated_test_type)
+    fail_tests = False
+    if isolated_test_types:
+        with ci_group(f"Isolated Testing {' '.join(escaped_tests)}"):
+            for isolated_test_type in isolated_test_types:
```

Comment:
```suggestion
```suggestion
    isolated_test_types = [isolated_test_type for isolated_test_type in ISOLATED_TEST_CASES if isolated_test_type in tests_to_run]            
    fail_tests = False
    if isolated_test_types:
        with ci_group(f"Isolated Testing {' '.join(escaped_tests)}"):
            for isolated_test_type in isolated_test_types:
                tests_to_run.remove(isolated_test_type)
```
```

Reply:
I decided to do it differently (separate steps in CI) - much nicer looking and generally simpler (I just had to add "excluded test list".